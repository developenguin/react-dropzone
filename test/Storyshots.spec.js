import initStoryshots from '@storybook/addon-storyshots';
import path from 'path';
import { render } from '@testing-library/react';


const reactTestingLibrarySerializer = {
  print: (val, serialize, indent) => serialize(val.container.firstChild),
  test: (val) => val && val.hasOwnProperty('container')
};

initStoryshots({
  configPath: path.join(__dirname, '..', '.storybook'),
  framework: 'react',
  renderer: render,
  snapshotSerializers: [reactTestingLibrarySerializer],
  storyKindRegex: /^((?!.*?DontTest).)*$/
});
