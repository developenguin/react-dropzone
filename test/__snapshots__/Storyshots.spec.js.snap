// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Dropzone With Custom Styling 1`] = `
<div
  class="dz-container container customContainer "
>
  <input
    class="dz-input input"
    multiple=""
    type="file"
  />
  <div
    class="dz-drag-files dragFiles customDragFiles"
  >
    <div
      class="previewWrapper"
    />
    <p
      class="dz-label label"
    >
      This one has custom styling.
    </p>
  </div>
</div>
`;

exports[`Storyshots Dropzone With Defaults 1`] = `
<div
  class="dz-container container "
>
  <input
    class="dz-input input"
    multiple=""
    type="file"
  />
  <div
    class="dz-drag-files dragFiles"
  >
    <div
      class="previewWrapper"
    />
    <p
      class="dz-label label"
    >
      Click or drag to upload.
    </p>
  </div>
</div>
`;
